api/src/main/java/org/apache/iceberg/expressions/Zorder.java

```diff
@@ -17,24 +17,19 @@
  * under the License.
  */
 
-package org.apache.spark.sql.catalyst.plans.logical
+package org.apache.iceberg.expressions;
 
-import org.apache.iceberg.NullOrder
-import org.apache.iceberg.Schema
-import org.apache.iceberg.SortDirection
-import org.apache.iceberg.SortOrder
-import org.apache.iceberg.expressions.Term
+import java.util.Arrays;
+import java.util.List;
 
-class SortOrderParserUtil {
+public class Zorder implements Term {
```

Comment:
should this implement `UnboundTerm`?

Because `SortOrder` only accepts `UnboundTerm`. https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/SortOrder.java#L258

Also, Now `sortOrder` needs some extra array or list that says which of the fields are Zordered in the sortOrder?

Reply:
No. `UnboundTerm` expects to have a single ref.